### PR TITLE
Fixing links

### PR DESCRIPTION
**Pull request by jaret-basden**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

- **docs\command-line-interface.md**: Updated a link so it is an absolute link that points to the Redocly output. Currently, it is a relative link that points to the GitHub-preview Markdown file for the content.
- **docs/entities.md**: Updated some links so they use an absolute link. Currently, they use a relative link, which doesn't work with the Redocly output (broken link).
- **docs/overview.md**: Updated some links so they they point to an id attribute to align with the Redocly output. Currently, they point to a raw .md file in the /docs folder. The Redocly output combines .md files in the /docs folder into a single index.html file.
- **docs/quickstart.md**: 
   - Updated some links so they they point to an id attribute to align with the Redocly output. Currently, they point to a raw .md file in the /docs folder. The Redocly output combines .md files in the /docs folder into a single index.html file.
   - Updated ./command-line-interface.md links to https://github.com/polaris-catalog/polaris/blob/main/docs/command-line-interface.md. This exposes a Copy button and presents the GitHub ToC on the left. 

Fixes # (issue): N/A

## Type of change

Please delete options that are not relevant.

- [ ] This change requires a documentation update

# How Has This Been Tested?

- [x] I tested the updated links locally in my browser (Chrome).

# Checklist:

Please delete options that are not relevant.

- [x] I have performed a self-review of my code
- [x] My changes generate no new warnings
- [x] I have signed and submitted the [ICLA](https://github.com/polaris-catalog/polaris/blob/main/ICLA.md) and if needed, the [CCLA](https://github.com/polaris-catalog/polaris/blob/main/CCLA.md). See [Contributing](https://github.com/polaris-catalog/polaris/blob/main/CONTRIBUTING.md) for details.

### CODE REVIEW

File: docs/command-line-interface.md [hunk 1]
```diff
@@ -797,7 +797,7 @@ polaris namespaces list --catalog my_catalog --parent a.b
 
 ### Privileges
 
-The `privileges` command is used to grant various privileges to a catalog role, or to revoke those privileges. Privileges can be on the level of a catalog, a namespace, a table, or a view. For more information on privileges, please refer to the [docs](./entities.md#privilege).
+The `privileges` command is used to grant various privileges to a catalog role, or to revoke those privileges. Privileges can be on the level of a catalog, a namespace, a table, or a view. For more information on privileges, please refer to the [docs](https://polaris.io/#tag/Polaris-Catalog-Entities/Privilege).
```
Comment (eric-maynard) [inline docs/command-line-interface.md:800]: Do we want the docs to point to internal URLs like this? That makes them not work offline. One alternative is to use local HTML docs like Apache Spark does ([e.g.](https://github.com/apache/spark/tree/master/docs#prerequisites))

**Review by collado-mike — APPROVED** [review]

**Review by takidau — APPROVED** [review]

**Comment by RussellSpitzer** [comment]
@jaret-basden Could you please rebase? We seem to have gotten a bit out of sync

**Comment by jaret-basden** [comment]
Hi Russell @RussellSpitzer , thank you for the feedback. I rebased and then opened new [PR # 231](https://github.com/apache/polaris/pull/231) to fix the links.

**Comment by flyrain** [comment]
Hi @jaret-basden, looks like there are still conflicts. Could you please rebase?

**Review by aihuaxu — APPROVED** [review]

**Comment by RussellSpitzer** [comment]
Closed in favor of https://github.com/apache/polaris/pull/231

**Pull request closed without merging**